api/spec/src/productcatalog/plan.tsp

```diff
@@ -71,81 +76,27 @@ model Plan {
  */
 @friendlyName("PlanPhase")
 model PlanPhase {
-  ...OmitProperties<UniqueResource, "id">;
+  ...SubscriptionPhaseCreateInput;
```

Comment:
Honestly I do not get this. Why the subscription models cannot reuse teh models defiend in plan? The subscription relies on plan and not the other way around. So I would rather put all the plan related fields and adjust subscription models accordingly. I am ok with changing field names, types, but not comfortable with pulling subscription models into plan. It feels super wierd and a good recept for disaster (e.g. circular dependency, etc.).

Reply:
In other words, plan models should be independent from subscription although synergies must be there for sure.